### PR TITLE
Feat: Adds tokens to `CartItem`

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

This PR applies the new [Theming structure](https://github.com/vtex-sites/base.store/pull/407) to `CartItem`: new scoped tokens linked to global ones.

## How does it work?

This PR uses local variables to parameterize the `CartItem` properties, then connects these scoped tokens to the global ones.

### New local tokens for `CartItem`

| Local token                                 | Default value/Global token linked          |
| - | - |
|`--fs-cart-item-padding`               | `var(--fs-spacing-3)`|
|`--fs-cart-item-bkg-color`             | `var(--fs-control-bkg)`|
|`--fs-cart-item-border-width`          | `var(--fs-border-width)`|
|`--fs-cart-item-border-color`          | `var(--fs-border-color-light)`|
|`--fs-cart-item-border-radius`         | `var(--fs-border-radius)`|

### Image

| Local token                                 | Default value/Global token linked          |
| - | - |
|`--fs-cart-item-image-border-radius`   | `var(--fs-cart-item-border-radius)`|

### Title

| Local token                                 | Default value/Global token linked          |
| - | - |
|`--fs-cart-item-title-margin-bottom`   | `var(--fs-spacing-0)`|
|`--fs-cart-item-title-line-height`     | `1.2`|
|`--fs-cart-item-title-color`           | `var(--fs-color-text)`|

### Actions

| Local token                                 | Default value/Global token linked          |
| - | - |
|`--fs-cart-item-actions-margin-top`   | `var(--fs-spacing-3)`|

## How to test it?

- The `CartItem` component and its variants should look just as it was before these changes.
- You can check the story on Storybook.

## Checklist

<em>You may erase this after checking them all ;)</em>

**Changelog**
- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.
- [x] Identified the function or parameter in the PR *- If applicable*. E.g., *`useWindowDimensions` hook*.

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*.
- [x] For documentation changes, ping @ carolinamenezes, @ PedroAntunesCosta or @ Mariana-Caetano to review and update.

### CODE REVIEW

File: src/components/cart/CartItem/cart-item.module.scss [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  // Default properties
+  --fs-cart-item-padding               : var(--fs-spacing-3);
+  --fs-cart-item-bkg-color             : var(--fs-color-neutral-0);
+
+  --fs-cart-item-border-width          : var(--fs-border-width);
+  --fs-cart-item-border-color          : var(--fs-border-color-light);
+  --fs-cart-item-border-radius         : var(--fs-border-radius);
+
+  // Image
+  --fs-cart-item-image-border-radius   : var(--fs-border-radius);
```
Comment (renatamottam) [inline src/components/cart/CartItem/cart-item.module.scss:17]:
```suggestion
  --fs-cart-item-image-border-radius   : var(--fs-cart-item-border-radius);
```

File: src/components/cart/CartItem/cart-item.module.scss [hunk 2]
```diff
@@ -0,0 +1,70 @@
+@import "src/styles/scaffold";
+
+.fs-cart-item {
+  // --------------------------------------------------------
+  // Design Tokens for CartItem
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-cart-item-padding               : var(--fs-spacing-3);
+  --fs-cart-item-bkg-color             : var(--fs-color-neutral-0);
```
Comment (renatamottam) [inline src/components/cart/CartItem/cart-item.module.scss:10]:
```suggestion
  --fs-cart-item-bkg-color             : var(--fs-control-bkg);
```

**Review by renatamottam — APPROVED** [review]
Nice 👏 
Left a couple improvement suggestions

File: src/components/cart/CartItem/cart-item.module.scss [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+  // Image
+  --fs-cart-item-image-border-radius   : var(--fs-border-radius);
+
+  // Title
+  --fs-cart-item-title-margin-bottom   : var(--fs-spacing-0);
+  --fs-cart-item-title-line-height     : 1.2;
+  --fs-cart-item-title-color           : inherit;
+  --fs-cart-item-title-text-decoration : none;
+  --fs-cart-item-title-outline         : none;
```
[the remark this answers is not in the document]
Comment (eduardoformiga) [reply src/components/cart/CartItem/cart-item.module.scss]: Thanks! done [9b6d770](https://github.com/vtex-sites/nextjs.store/pull/102/commits/9b6d7705a50d17571c3d11f80decfca701385d83)

File: src/components/cart/CartItem/cart-item.module.scss (hunk 1 above)
Comment (eduardoformiga) [reply src/components/cart/CartItem/cart-item.module.scss]: done [d6c8cf1](https://github.com/vtex-sites/nextjs.store/pull/102/commits/d6c8cf18eb9a3296ccbfecee50c21e926fd9e7e4)

File: src/components/cart/CartItem/cart-item.module.scss (hunk 2 above)
Comment (eduardoformiga) [reply src/components/cart/CartItem/cart-item.module.scss]: done [bec82b7](https://github.com/vtex-sites/nextjs.store/pull/102/commits/bec82b7f9cc7c8a2c62d581d39a6f7fdbc41d840).

File: src/components/cart/CartItem/cart-item.module.scss [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  --fs-cart-item-border-radius         : var(--fs-border-radius);
+
+  // Image
+  --fs-cart-item-image-border-radius   : var(--fs-cart-item-border-radius);
+
+  // Title
+  --fs-cart-item-title-margin-bottom   : var(--fs-spacing-0);
+  --fs-cart-item-title-line-height     : 1.2;
+  --fs-cart-item-title-color           : var(--fs-color-text);
+  --fs-cart-item-title-text-decoration : none;
```
Comment (hellofanny) [inline src/components/cart/CartItem/cart-item.module.scss:23]: Do we need this one? 🤔

**Review by hellofanny — APPROVED** [review]
LGTM! ⭐

File: src/components/cart/CartItem/cart-item.module.scss (hunk 4 above)
Comment (eduardoformiga) [reply src/components/cart/CartItem/cart-item.module.scss]: not sure, I'm gonna remove it then.
Done [9cba310](https://github.com/vtex-sites/nextjs.store/pull/102/commits/9cba310050da16f4aa508b27208ae6275f66997d).

**Review by lucasfp13 — APPROVED** [review]
LGTM!

**Pull request merged**